### DISCUSSION TITLE
Show month name with prevLabel and nextLabel

### DISCUSSION THREAD

**Discussion by annsusan** [wojtekmaj/react-calendar#645] [Q&A] [answered]
Hi,

I would like to show previous month name with prevLabel and next month name with nextLabel. Also this should update to new month name while clicking.

![custom](https://user-images.githubusercontent.com/4178204/190067771-98ca2587-1820-4a0c-aa7c-210a032708a6.png)

**Comment by wojtekmaj** [comment, marked as answer]
You can! 

* Make `activeStartDate` and `view` controlled.
* Use these values to calculate previous and next labels.
* Pass the labels to Calendar as `prevLabel` and `nextLabel`

https://codesandbox.io/s/react-calendar-prev-next-labels-pmytut

**Reply by annsusan** [reply]
Thanks for the quick reply.
